syntax/hive.vim

```diff
@@ -70,10 +70,11 @@ syn keyword sqlType array map struct named_struct create_union timestamp date va
 
 syn match hiveVar     "hive\.[a-zA-Z.]\+"
 syn match hiveVar     "mapred\(uce\)\=\.[a-zA-Z.]\+"
+syn match hiveVar2    "\${[^}]\+}"
```

Comment:
Could we use a more descriptive name for this? `hiveVarSubst` maybe?

Reply:
good suggestion!